{{cookiecutter.github_repository}}/circle.yml

```diff
@@ -0,0 +1,87 @@
+## Customize the test machine
+machine:
+  python:
+    version: {{ '3.5' if cookiecutter.python3.lower() == 'y' else '2.7.4'}} # 2.7 will cause an error with py.test
+  environment:
+    # each command runs in its own shell, so just adding an export FOO=bar command won't work
+    DATABASE_URL: postgres://postgres@localhost/{{ cookiecutter.main_module }}
+  #services: # By default popular ones are running on our build machines (bound to localhost), including Postgres, MySQL, Redis and MongoDB
+  # -redis 
+
+## Customize checkout
+#checkout:
+#  post:
+#	   - git submodule sync
+
+## Customize dependencies
+dependencies:
+  override:
+    - pip install pip-accel
+    - pip-accel install ansible
+    - pip-accel install -r requirements/development.txt
```

Comment:
You can remove the use of `pip-accel` on cirlce ci, it caches and restores the virtualenv by-default, only new dependencies are installed in the subsequent builds.